src/components/TopBar/TopBar.vue

```diff
@@ -235,6 +248,37 @@ export default {
 		},
 	},
 
+	watch: {
+		unreadMessagesCounter(newValue, oldValue) {
+			if (!this.isInCall || !this.showOpenSidebarButton) {
+				return
+			}
+
+			// new messages arrived
+			if (newValue > 0 && oldValue === 0) {
+				if (this.hasUnreadMentions) {
+					showMessage(t('spreed', 'You have been mentioned in the conversation sidebar.'))
+				} else {
+					showMessage(t('spreed', 'You have new unread messages in the conversation sidebar.'))
+				}
+			}
+		},
+
+		hasUnreadMentions(newValue, oldValue) {
+			if (!this.isInCall || !this.showOpenSidebarButton) {
+				return
+			}
+
+			// in some cases the user already had unread messages and saw the other notification
+			// but then got mentionned
+			// the limit of "> 1" is here to avoid seeing two notifications, one for the first non-mention
+			// unread message and the second for the mention
+			if (newValue && this.unreadMessagesCounter > 1) {
+				showMessage(t('spreed', 'You have been mentioned in the conversation sidebar.'))
```

Comment:
```suggestion
				showMessage(t('spreed', 'You have been mentioned in the chat.'))
```

Reply:
I've been avoiding the term "chat" everywhere, believing that we had settled to only use the term "conversation" in visible texts ?

Reply:
No, we avoid "room" and use conversation instead.
But the chat is the chat. Also refering to the sidebar withit is weird, as when you leave the call and try to look at the sidebar where you have been mentioned you wont find anything as the chat is then in the main view